cmd/vic-machine/create/create.go

```diff
@@ -543,6 +550,10 @@ func (c *Create) processParams() error {
 		return err
 	}
 
+	if err := c.processDefaultVolumeStore(); err != nil {
+		return errors.Errorf("Error occurred while processing the default volume stores: %s", err)
```

Comment:
I'm not sure what this error is meant to convey.  What does "process" mean?  And looking at the function itself, it returns nil for error exclusively.  So this error will never be thrown.